### PR TITLE
Add Rubocop to CI

### PR DESCRIPTION
**Pull request by jekyllbot**
PR automatically created for @pathawks.

```text
Add Rubocop to CI
```

### CODE REVIEW

File: jekyll-coffeescript.gemspec [hunk 1]
```diff
@@ -21,4 +21,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.5"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec"
+  spec.add_development_dependency "rubocop"
```
Comment (pathawks) [inline jekyll-coffeescript.gemspec]: Should probably limit version here.

**Comment by pathawks** [comment]
Travis can't find Jekyll's `.rubocop.yml`

File: jekyll-coffeescript.gemspec (hunk 1 above)
Comment (DirtyF) [reply jekyll-coffeescript.gemspec]: `"rubocop", "~> 0.5"` ?

**Comment by DirtyF** [comment]
@pathawks why do we still test against Jekyll 2.5?

**Comment by pathawks** [comment]
@DirtyF Because nobody has updated `.travis.yml` 😆

**Comment by DirtyF** [comment]
@pathawks done in #22

**Comment by DirtyF** [comment]
@jekyllbot: merge +dev

**Pull request merged**